internal/backends/winit/accesskit.rs

```diff
@@ -487,26 +487,34 @@ impl NodeCollection {
         }
 
         if item
-            .accessible_string_property(AccessibleStringProperty::Selectable)
+            .accessible_string_property(AccessibleStringProperty::ItemSelectable)
             .is_some_and(|x| x == "true")
         {
             node.set_selected(
-                item.accessible_string_property(AccessibleStringProperty::Selected)
+                item.accessible_string_property(AccessibleStringProperty::ItemSelected)
                     .is_some_and(|x| x == "true"),
             );
         }
 
         if let Some(position_in_set) = item
-            .accessible_string_property(AccessibleStringProperty::PositionInSet)
+            .accessible_string_property(AccessibleStringProperty::ItemIndex)
             .and_then(|s| s.parse::<usize>().ok())
         {
             node.set_position_in_set(position_in_set);
-        }
-        if let Some(size_of_set) = item
-            .accessible_string_property(AccessibleStringProperty::SizeOfSet)
-            .and_then(|s| s.parse::<usize>().ok())
-        {
-            node.set_size_of_set(size_of_set);
+            let mut item = item.clone();
```

Comment:
We agree to set `size_of_set` on the container node rather than each item. It will be done as part of my ongoing work on listboxes so you can revert most of your changes here. BTW, the parent selection below would not be enough as listbox items can be inside generic containers, which can then be children of the listbox.